mlrun/projects/project.py

```diff
@@ -2884,6 +2885,23 @@ def _init_function_from_obj(func, project, name=None):
     return name or func.metadata.name, func
 
 
+def _mask_credentials_in_function_object(function):
+    for env in function.spec.env:
```

Comment:
As mentioned in Slack discussion, should also look for access-key in volumes (which may be added as part of fuse-mount)